model/fieldtypes/compat/Float.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
```

Comment:
@kinglozzer @dhensby what do you think of this approach? I've added a separate autoloader for the Int/Float classes so as not to pollute manifest: https://github.com/silverstripe/silverstripe-framework/pull/6771/commits/c3ed81b9df0e278b85143fff59836b41ce3cdbb4